ddtrace/tracer/span.go

```diff
@@ -912,6 +913,28 @@ func (s *Span) AddEvent(name string, opts ...SpanEventOption) {
 	s.spanEvents = append(s.spanEvents, event)
 }
 
+// RecordException attaches error information to the current span as a span event.
+// Additional span event attributes can be provided. The values may be (array of)
+// string, boolean, int or float type.
+func (s *Span) RecordException(err error, attributes ...map[string]any) {
+	if err == nil {
+		return
+	}
+
+	eventAttributes := map[string]any{
+		"exception.type":       reflect.TypeOf(err).String(),
+		"exception.message":    err.Error(),
+		"exception.stacktrace": takeStacktrace(0, 0),
+	}
+
+	// Merge with provided attributes if any, with provided attributes taking precedence
+	if len(attributes) > 0 {
+		maps.Copy(eventAttributes, attributes[0])
+	}
```

Comment:
You are only looking at the first in the slice of `attributes` here, what about the rest of the elements?
Could change this into a `for range` loop, iterating over the length of `attributes`, and just omit the check on length.
```suggestion
		for _, attrMap := range attributes {
    maps.Copy(eventAttributes, attrMap)
}